# HG changeset patch
# User Pantxo Diribarne <pantxo.diribarne@gmail.com>
# Date 1422886402 -3600
#      Mon Feb 02 15:13:22 2015 +0100
# Node ID 50f14692e33b7838fc201c006a4e12513df83a5d
# Parent  830c8457295a6f0b5e2ca9797633689101877db4
Freset: do not reset figure outerposition (part of bug #43960)

* graphics.cc: erase "outerposition" from the list of factory defaults.

diff --git a/libinterp/corefcn/graphics.cc b/libinterp/corefcn/graphics.cc
--- a/libinterp/corefcn/graphics.cc
+++ b/libinterp/corefcn/graphics.cc
@@ -4533,16 +4533,17 @@ void
 figure::reset_default_properties (void)
 {
   // empty list of local defaults
   default_properties = property_list ();
 
   property_list::pval_map_type plist = xproperties.factory_defaults ();
   plist.erase ("units");
   plist.erase ("position");
+  plist.erase ("outerposition");
   plist.erase ("paperunits");
   plist.erase ("paperposition");
   plist.erase ("windowstyle");
   xreset_default_properties (get_handle (), plist);
 
   // FIXME: the following short sleep is needed in order
   //        to avoid a crash when using qt toolkit
   Fsleep (octave_value (0.001));
